app/assets/javascripts/discourse/components/topic-footer-buttons.js.es6

```diff
@@ -46,6 +40,7 @@ export default Component.extend({
 
   @discourseComputed
   showAdminButton() {
+    return true;
```

Comment:
If `showAdminButton` is always true it probably doesn't need to exist does it?

Reply:
woops! temporary override made its way in